cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-deployment.adoc

```diff
@@ -0,0 +1,153 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-deployment"]
+= Tutorial: Deploying an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-deployment
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 23-JAN-2024
+
+== Deploying the OSToy application with Kubernetes
```

Comment:
Do we need to set a `Prerequisites` section right before the deployment tutorial?
Examples:
1. AWS account setup if required
2. An available ROSA cluster deployed and available
3. Set env vars if needed etc..

Reply:
Makes sense, I will add the ROSA cluster one. I would assume your AWS account is already setup if you have a ROSA cluster deployed.